cpp/src/io/parquet/writer_impl.cu

```diff
@@ -455,7 +465,7 @@ struct leaf_schema_fn {
     col_schema.type           = Type::INT32;
```

Comment:
`timestamp_D` should correspond to `DateType` in arrow, though there is no way in cuDF to actually create `timestamp_D` columns.